src/gridstack-dd.ts

```diff
@@ -267,16 +265,24 @@ GridStack.prototype._setupAcceptWidget = function(this: GridStack): GridStack {
         GridStackDD.get().remove(el);
       }
       if (!wasAdded) return false;
+
       el.gridstackNode = node;
       node.el = el;
-      // @ts-ignore
-      Utils.copyPos(node, this._readAttr(this.placeholder)); // placeholder values as moving VERY fast can throw things off #1578
-      Utils.removePositioningStyles(el);// @ts-ignore
+      // placeholder values as moving VERY fast can throw things off #1578
+      Utils.copyPos(node, this._readAttr(this.placeholder));
+      Utils.removePositioningStyles(el);
       this._writeAttr(el, node);
-      this.el.appendChild(el);// @ts-ignore
+      this.el.appendChild(el);
       this._updateContainerHeight();
-      this.engine.addedNodes.push(node);// @ts-ignore
-      this._triggerAddEvent();// @ts-ignore
+
+      const subgrid = node.subGrid as GridStack;

```

Comment:
that looks a lot better. Just add this comment
// if dropping a subgrid, make sure to update its style #1943